tests/unittests/test_pep8.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+# ###################################################
+# Copyright (C) 2008-2016 The Unknown Horizons Team
+# team@unknown-horizons.org
+# This file is part of Unknown Horizons.
+#
+# Unknown Horizons is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the
+# Free Software Foundation, Inc.,
+# 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+# ###################################################
+
+import os
+from unittest import TestCase
+
+import pep8
+
+
+class TestCodeFormat(TestCase):
```

Comment:
I don't think this should be part of the test suite, but instead be integrated into the CI flow on travis. The configuration you added in the test can probably go into either setup.cfg or .pep.rc or whatever it's called, and contributers can then run `pep8` to run the style checks.